Pascal_Triangle/Pascal_Triangle.py

```diff
@@ -0,0 +1,20 @@
+def Pascal_Triangle(n):  
+    for row in range(1, n + 1):  
+        # Initializing the Initial Value of Pascal Value as 1
+        pascal_value = 1
+        # For printing blank spaces
+        for i in range(1, n - row + 1):
+            print(' ',end=' ') 
+        # For calculating the pascal value 
+        for i in range(1, row + 1): 
+            # Creating the printable string and printing the string
+            s = ' ' + str(pascal_value) + ' '
+            print(s, end = " ")  
+            # Calculating the new Pascal Value
+            pascal_value = int(pascal_value * (row - i) / i) 
+        print(end='\n')  
```

Comment:
Please add space around the operator

Reply:
I have already done it , just take a look .

Reply:
it should be print(end = '\n') not print(end='\n')

Reply:
ok